device-types/Fortinet/FG-300C.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 300C
+slug: fg-300c
+part_number: FG-300C
+u_height: 1
+is_full_depth: false
```

Comment:
Please add `airflow` if you have it

Reply:
Can't find this info

Reply:
Well after looking at the photo it must be front-to-rear. I'll add this info.